apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -106,6 +107,31 @@ const useStyles = makeStyles((theme) => ({
     paddingLeft: '8px',
     paddingRight: '8px',
   },
+  barContainer: {
+    position: 'relative',
+  },
+  bar: {
+    border: '1px solid',
+    borderColor: '#D9D9D9',
+    height: '57%',
+    position: 'absolute',
+    zIndex: '-1',
+    alignSelf: 'center',
+    left: '1%',
```

Comment:
Is `%` (percent) intentional? Otherwise, I would have expected `em/rem` or even `px`.

Reply:
Will fix :)